collect_app/src/main/java/org/odk/collect/android/projects/QrCodeProjectCreatorDialog.kt

```diff
@@ -144,6 +148,13 @@ class QrCodeProjectCreatorDialog :
             dismiss()
         }
         beepManager = BeepManager(requireActivity())
+
+        return binding.root
+    }
+
+    override fun onStart() {
```

Comment:
Moving `startScanning` to `onStart` needs a test.

Reply:
Done.